Meshtastic/Resources/docs/user/signal-meter.html

```
@@ -28,48 +28,52 @@ <h2>2. The Magic of LoRa: Hearing &quot;Below the Noise Floor&quot;</h2>
 <td>Transmission completely buried in static.</td>
 </tr>
 </tbody>
 </table>
+<p>RSSI is still shown next to the meter, but it is no longer used to guess the rating with fixed thresholds — on its own, RSSI can't tell you how much of that power is signal versus noise.</p>
+<p><strong>Using the real noise floor.</strong> When your receiving radio has recently reported its own <strong>noise floor</strong> (part of its Local Stats telemetry), the app does one better: it computes your true link margin as <code>RSSI − noise floor</code> and checks that against the same preset limit too, then shows the <strong>more conservative</strong> of the two ratings. This catches the case where the reported SNR looks fine but local interference is quietly eating your margin. When no recent noise floor is available, the app simply uses the SNR-only rating above.</p>
 <hr />
 <h2>4. What This Means for You</h2>
 <p>Because Meshtastic's meter acts as a <strong>&quot;Clarity Meter&quot;</strong>, it behaves differently than what most people expect:</p>
-<div class="tips-callout"><strong>Tip — Don't panic over low RSSI:</strong> You might see a seemingly terrible RSSI value like <code>-118 dBm</code>. On a cell phone, you would have zero bars. But if you have an SNR of <code>+2 dB</code>, Meshtastic will still show a strong signal! <em>The library is quiet, so the whisper is heard perfectly.</em></div>
-<div class="warning-callout"><strong>Warning — Watch out for local noise:</strong> If you hook up a massive antenna and see a great RSSI (e.g., <code>-90 dBm</code>) but your signal meter is only showing <strong>1 Bar (Bad)</strong>, you have a problem. It means you have local interference—perhaps a cheap power supply, a noisy computer, or a nearby radio tower—creating so much static that it is drowning out your mesh.</div>
+<div class="tips-callout"><strong>Tip — Don't panic over low RSSI</strong></p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)